### PR TITLE
Patch for RHEL-07-010050

### PR DESCRIPTION
**Pull request by chrisruffalo**
- added "Banner /etc/isssue" to /etc/ssh/sshd_conf
 - set appropriate text in /etc/issue

### CODE REVIEW

File: tasks/fix-cat2.yml [hunk 1]
```diff
@@ -15,13 +15,31 @@
...
+    owner: root
+    group: root
+    mode: 644
   tags:
       - cat2
       - medium
       - patch
       - RHEL-07-010050
 
+- lineinfile:
```
Comment (johannes-cabal) [inline tasks/fix-cat2.yml]: Please Add:
 - name: "MEDIUM | RHEL-07-010050 | PATCH | The operating system must display the Standard Mandatory DoD Notice and Consent Banner before granting local or remote access to the system via a command line user logon."

I know it's redundant, but that's the way we've decided to go. Want to make sure any user knows specifically what is being accomplished as the task is run (i.e. if they were to use step-mode, they can track)

File: tasks/fix-cat2.yml [hunk 2]
```diff
@@ -15,13 +15,31 @@
       - RHEL-07-010040
 
 - name: "MEDIUM | RHEL-07-010050 | PATCH | The operating system must display the Standard Mandatory DoD Notice and Consent Banner before granting local or remote access to the system via a command line user logon."
-  command: "true"
+  copy:
+    src: "rhel_07_010050_issue_text"
```
Comment (johannes-cabal) [inline tasks/fix-cat2.yml]: Please make block notation 4 spaces.

File: tasks/fix-cat2.yml [hunk 3]
```diff
@@ -15,13 +15,32 @@
       - RHEL-07-010040
 
 - name: "MEDIUM | RHEL-07-010050 | PATCH | The operating system must display the Standard Mandatory DoD Notice and Consent Banner before granting local or remote access to the system via a command line user logon."
-  command: "true"
+  copy:
```
Comment (josh-springer) [inline tasks/fix-cat2.yml]: See 010119 as an example.

File: tasks/fix-cat2.yml [hunk 4]
```diff
@@ -15,13 +15,32 @@
...
+        regexp: ^#?Banner
+        line: Banner /etc/issue
+        validate: sshd -tf %s
+  notify: restart sshd
+  tags:
+      - cat2
+      - medium
+      - patch
+      - RHEL-07-010050      
+
```
Comment (johannes-cabal) [inline tasks/fix-cat2.yml]: Add tag
- ssh

File: tasks/fix-cat2.yml [hunk 5]
```diff
@@ -15,13 +15,32 @@
...
+        src: "rhel_07_010050_issue_text"
+        dest: "/etc/issue"
+        owner: root
+        group: root
+        mode: 644
   tags:
       - cat2
       - medium
       - patch
       - RHEL-07-010050
```
Comment (josh-springer) [inline tasks/fix-cat2.yml]: Add tag
- ssh

Comment (johannes-cabal) [inline tasks/fix-cat2.yml]: Add tag
- ssh

File: tasks/fix-cat2.yml [hunk 6]
```diff
@@ -15,12 +15,32 @@
       - RHEL-07-010040
 
 - name: "MEDIUM | RHEL-07-010050 | PATCH | The operating system must display the Standard Mandatory DoD Notice and Consent Banner before granting local or remote access to the system via a command line user logon."
-  command: "true"
+  copy:
+        src: "rhel_07_010050_issue_text"
```
Comment (johannes-cabal) [inline tasks/fix-cat2.yml]: Space module args 4 spaces instead of 6 spaces.

**Review by johannes-cabal — APPROVED** [review]
Everything looks good-to-go. Appropriate tags and spacing issues were corrected.

**Pull request merged**